src/Components/WebAssembly/WebAssembly.Authentication/src/Services/AuthorizationMessageHandler.cs

```diff
@@ -16,10 +13,11 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
     /// Access tokens will only be added when the request URI is within one of the base addresses configured using
     /// <see cref="ConfigureHandler(IEnumerable{string}, IEnumerable{string}, string)"/>.
     /// </summary>
-    public class AuthorizationMessageHandler : DelegatingHandler
+    public class AuthorizationMessageHandler : DelegatingHandler, IDisposable
```

Comment:
[`HttpMessageHandler`](https://github.com/dotnet/runtime/blob/91f43669bbd6bdf1ba760a6eaa4e42c2794c5f08/src/libraries/System.Net.Http/src/System/Net/Http/HttpMessageHandler.cs#L12) from which [`DelegatingHandler`](https://github.com/dotnet/runtime/blob/91f43669bbd6bdf1ba760a6eaa4e42c2794c5f08/src/libraries/System.Net.Http/src/System/Net/Http/DelegatingHandler.cs#L13) inherits already implements `IDisposable` so you could just override the existing [`protected virtual void Dispose(bool disposing)`](https://github.com/dotnet/runtime/blob/91f43669bbd6bdf1ba760a6eaa4e42c2794c5f08/src/libraries/System.Net.Http/src/System/Net/Http/HttpMessageHandler.cs#L31) method.

Reply:
That's what I'd originally done prior to https://github.com/dotnet/aspnetcore/pull/36358/commits/e2351d3df4c1fb12eeeeaa1d3559f40392a1fc2d. The issue with that approach is it requires adding a new public API, which at this stage of the release is a bit tougher.